config/src/client.rs

```diff
@@ -60,7 +60,7 @@ pub struct BasicAuth {
 }
 
 /// `Configuration` provides an ability to define client parameters such as `TORII_URL`.
-#[derive(Debug, Clone, Deserialize, Serialize, Proxy, LoadFromEnv, Documented)]
+#[derive(Debug, Clone, Deserialize, Serialize, Proxy, LoadFromEnv, Documented, PartialEq, Eq)]
 #[serde(rename_all = "UPPERCASE")]
 #[config(env_prefix = "IROHA_")]
 pub struct Configuration {
```

Comment:
I think we could add here `finalize` method as well and in finalize we would check configuration invaraints, log warnings and so on.

Reply:
I propose a different naming convention, one that avoids the `ize`/`ise` suffices.